### DISCUSSION TITLE
Docker for InfluxDB V2

### DISCUSSION THREAD

**Discussion by l3g3ndury** [jertel/vuegraf#92] [Q&A] [answered]
I am quite new to python and pre-developed scripts, so if I sound a bit ... noobish, I apologize. 

But for my research, I am supposed to gather information using InfluxDB2 with Emporia. However, I have ran into an issue:

My first issue is that I got an error similar to that of discussion 87, which I will reference right here

`
 Traceback (most recent call last):
   File "c:\Program Files\InfluxData\vuegraf-master\src\vuegraf\vuegraf.py", line 207, in <module>
     signal.signal(signal.SIGHUP, handleExit)
 AttributeError: module 'signal' has no attribute 'SIGHUP'
`

However, as you did respond with using a docker container, I have installed the windows version. But, my first issue is that the commands you have given were using Influx V1, where my environment has already been accustomed to use Influx V2. _Is there any way to use the container with V2?_ 

Also, while trying to instantiate the first command of the docker, I got an error along the lines of 
`
error during connect: This error may indicate that the docker daemon is not running.: Post "http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/images/create?fromImage=jertel%2Fvuegraf&tag=latest": open //./pipe/docker_engine: The system cannot find the file specified.
`
I also did try using the "docker pull jertel/vuegraf" command but that isn't working as well. What I specifically did for installing vuegraf to my computer is by cloning/forking the repository onto my computer. _Is there anything visibly wrong with my setup?_

Thank you so much for being available to help.

**Comment by jertel** [comment, marked as answer]
For InfluxDB v2, see the project README, where it shows a configuration using InfluxDB v2: https://github.com/jertel/vuegraf

As far as the Docker error goes, I don't use Docker on Windows so I can't help much here. You will need to figure out why Docker isn't running before you can proceed with pulling and running the image. Perhaps search the Docker forums, Google, etc for the error you're encountering.

**Reply by l3g3ndury** [reply]
I apologize, but the explanation isn't as intuitive for me. What I read was doing a configuration for InfluxDB v1.8. However, I am using InfluxDB v2 and when I replaced the 1.8 with 2.0 or 2.1, it still doesn't work. Is there a more specific command for v2.0?

**Reply by jertel** [reply]
![image](https://user-images.githubusercontent.com/16639611/158496125-966ca145-74ef-4f0c-9c28-3ce113fe8213.png)
